### PR TITLE
[BTS-1350]

### PR DESCRIPTION
**Pull request by cpjulia**
### Scope & Purpose

 Fixed imprecision in index info in telemetrics object for cluster. When the collection also had an arangosearch view, telemetrics showed imprecise index values because the view was being considered as an index of type arangosearch, and its object wouldn't contain expected fields that would be found in other indexes' objects.

- [x] :hankey: Bugfix
- [ ] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [x] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [x] **integration tests**
  - [ ] **resilience tests**
- [ ] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports
  - [ ] Backport for 3.11: https://github.com/arangodb/arangodb/pull/18933

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/BTS-1350
- [ ] Design document:

### CODE REVIEW

File: tests/js/client/server_parameters/test-telemetrics-true-auth.js [hunk 1]
```diff
@@ -302,7 +309,7 @@ function telemetricsShellReconnectSmartGraphTestsuite() {
                 numColls++;
                 assertEqual(coll.idxs.length, 4);
               } else {
-                assertEqual(nDocs, 0);
+                assertTrue(nDocs === 0 || nDocs === 1);
```
Comment (jsteemann) [inline tests/js/client/server_parameters/test-telemetrics-true-auth.js:312]:
```suggestion
                assertTrue(nDocs <= 1, nDocs);
```

File: tests/js/client/server_parameters/test-telemetrics-true-auth.js [hunk 2]
```diff
@@ -433,7 +445,7 @@ function telemetricsShellReconnectGraphTestsuite() {
...
```
Comment (jsteemann) [inline tests/js/client/server_parameters/test-telemetrics-true-auth.js:448]:
```suggestion
                assertTrue(nDocs <= 1, nDocs);
```

File: tests/js/client/server_parameters/test-telemetrics-true-auth.js [hunk 3]
```diff
@@ -556,7 +574,7 @@ function telemetricsApiReconnectSmartGraphTestsuite() {
...
```
Comment (jsteemann) [inline tests/js/client/server_parameters/test-telemetrics-true-auth.js:577]:
```suggestion
                assertTrue(nDocs <= 1, nDocs);
```

File: tests/js/client/server_parameters/test-telemetrics-true-auth.js [hunk 4]
```diff
@@ -697,7 +721,7 @@ function telemetricsApiReconnectGraphTestsuite() {
...
```
Comment (jsteemann) [inline tests/js/client/server_parameters/test-telemetrics-true-auth.js:724]:
```suggestion
                assertTrue(nDocs <= 1, nDocs);
```

File: tests/js/client/server_parameters/test-telemetrics-true-auth.js [hunk 5]
```diff
@@ -799,7 +823,7 @@ function telemetricsEnhancingOurCalm() {
         let res = arango.GET_RAW("/_admin/telemetrics", { "user-agent": "arangosh/3.11.0" });
         if (res.code === 200) {
           ++successful;
-        } 
+        }
         if (res.code === 420) {
```
Comment (jsteemann) [inline tests/js/client/server_parameters/test-telemetrics-true-auth.js:827]: https://httpstatusdogs.com/420-enhance-your-calm

**Review by gnusi — CHANGES_REQUESTED** [review]
We need information about links as well

**Review by gnusi — APPROVED** [review]

**Review by jsteemann — APPROVED** [review]
LGTM

**Pull request merged**